addons/mail/static/src/discuss/core/common/avatar_stack.js

```diff
@@ -15,28 +15,37 @@ import { Component } from "@odoo/owl";
 export class AvatarStack extends Component {
     static template = "mail.AvatarStack";
     static props = {
+        containerClass: { type: String, optional: true },
         direction: { type: String, optional: true, validate: (d) => ["v", "h"].includes(d) },
         avatarClass: { type: Function, optional: true },
         max: { type: Number, optional: true },
+        onClick: { type: Function, optional: true },
         personas: Array,
         size: { type: Number, optional: true },
         slots: { optional: true },
     };
     static defaultProps = {
         avatarClass: () => "",
+        onClick: () => {},
         max: 4,
         size: 24,
         direction: "h",
     };
 
     getStyle(index) {
-        let style = `width: ${this.props.size}px; height: ${this.props.size}px;`;
-        if (index === 0) {
-            return style;
+        const paddingDirection = this.props.direction === "v" ? "bottom" : "right";
+        const styles = [
+            "box-sizing: content-box",
+            `height: ${this.props.size}px`,
+            `padding-${paddingDirection}: ${this.props.size / 15}px`,
```

Comment:
Directional padding is not elegant: the contour is ellipsis-shaped rather than a circle:
<img width="187" alt="Screenshot 2025-02-24 at 11 51 36" src="https://github.com/user-attachments/assets/28d2a18c-8f00-485c-be19-f1dd08020faa" />
<img width="187" alt="Screenshot 2025-02-24 at 11 51 36 copy" src="https://github.com/user-attachments/assets/c600801e-6678-4f17-af52-2f51a0311b81" />

I would put a 1px constant padding instead:
<img width="162" alt="Screenshot 2025-02-24 at 11 57 23" src="https://github.com/user-attachments/assets/f0827536-50a6-4ffa-b8a8-d5a287157ce7" />